databases/sql-2.md

```diff
@@ -45,24 +45,26 @@ Here's a sample schema:
         - which has an integer field named: `user_id`
         - and a text field named: `content`
 
-2.  Enter the SQLite command line program using the terminal commad `sqlite3 twitter.sqlite3`. (This will save your database to a file called `twitter.sqlite3`). Once there, use SQL statements to create two tables according to the spec for the two tables in step 1.
+2. Save your database by using the `command pg_dump twitter > twitter.psql`. (This will save your database to a file called `twitter.psql`.)
```

Comment:
As before, this should probably be a CREATE DATABASE command.

Reply:
If I understand correctly, you want them to use their DB named after their username that was created when installing Postgres for the table in these exercises? (The current commit reflects changes based on my understanding.)